packages/cli/lib/prompts.js

```diff
@@ -90,7 +90,7 @@ const ACCOUNT_NAME = {
 const ACCOUNT_API_KEY = {
   name: 'apiKey',
   message:
-    'Enter the API key for your account (found at https://app.hubspot.com/l/api-key):',
+    'Enter the API key for your account (found at https://app.hubspot.com/l/api-key ):',
```

Comment:
I wonder if there is a better way to fix this. Adding a space to the parentheses feels a bit off. @erincouse maybe you have a suggestion for how to reword?